user/invite-new.md

```diff
@@ -4,6 +4,32 @@ description: How to invite a new user
 
 # Invite user
 
-Explanation
+You can invite a new user to your company:
 
-API
+## Request
+
+{% api-method method="post" host="https://api.accp.tradecloud1.com/v2" path="/user"%} 
+{% api-method-summary %} Create user {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Authentication token {% endapi-method-parameter %} 
```

Comment:
Value is: `Bearer <Authentication token>`
Change in all pages

Reply:
Fixed